splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/DB2VarcharCompatibilityIT.java

```diff
@@ -325,4 +340,140 @@ public void testInvalidateStoredStatements() throws Exception {
         // Restore the flag setting to the value when this test started.
         methodWatcher.executeUpdate("call syscs_util.syscs_set_global_database_property('splice.db2.varchar.compatible', true)");
     }
+
+    @Test
+    public void testMultiProbeScan() throws Exception {
+        String sqlTemplate = "select * from t a --splice-properties useSpark=" + useSpark.toString() +
+        ", index=%s\n" +
+        "where v in ('SBVGCCC    ', 'G')\n" +
+        "and c in ( ' ', CAST('A' AS CHAR(2)))";
```

Comment:
this strings are weirdly intended. i think we should intend multi-line commands like e.g.
```
String sqlTemplate = "select * from t a --splice-properties useSpark=" + useSpark.toString() +
        ", index=%s\n" +
        "where v in ('SBVGCCC    ', 'G')\n" +
        "and c in ( ' ', CAST('A' AS CHAR(2)))";
```

Reply:
OK, I've fixed the indentation.